packages/custodyController/src/custody.ts

```diff
@@ -104,34 +104,6 @@ export class CustodyController {
     return custodyTypes;
   }
 
-  // TODO - Can probably be removed as that state is not being used anywhere
-  setCustodianConnectRequest({
-    token,
```

Comment:
I think setCustodianConnectRequest is being used in the extension. https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js#L79
Also getCustodianConnectRequest is being referenced there, but doesn't seem to be used in the `interactive-replacement-token-page.js`

Reply:
I removed them from the extension as we don't need them:
https://github.com/MetaMask/metamask-extension/pull/23000